index.html

```diff
@@ -4,14 +4,27 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" text="text/css" href="css/my_website.css">
     <title>Alejandro Ortiz</title>
 </head>
 
 <body>
-    <p>Hi, I’m @alejandro-ortiz <br>
-        I’m interested in Rocketry, Tennis, Web Development, Electronics <br>
+    <div class="main-header">
+        <h1 id="title">Alejandro Ortiz</h1>
+    </div>
+    <p class="text">Hi, I’m @alejandro-ortiz <br>
+        I’m interested in Rocketry, Tennis, Web Development, Electronics, and scouts<br>
         I’m currently learning Git and Javascript
```

Comment:
Might want to capitalize Scouts. Also, it's JavaScript (capital S).

Reply:
got it